requirements/default.txt

```diff
@@ -1 +1,5 @@
-decorator>=4.3.0
+decorator>=4.4
+numpy>=1.19; platform_python_implementation!='PyPy' and python_version<'3.9'
+scipy>=1.5; platform_python_implementation!='PyPy' and python_version<'3.9'
+matplotlib>=3.3; platform_python_implementation!='PyPy' and python_version<'3.9'
+pandas>=1.1; platform_python_implementation!='PyPy' and python_version<'3.9'
```

Comment:
I anticipate this would cause a bit of pain for a few projects which don't really need these

Reply:
What projects are you thinking of and what parts of networkx do they depend on?

Reply:
well one example is https://github.com/iterative/dvc, which just resorted to pinning networkx to version 2.5 specifically to avoid this potential issue. Your call though.